src/System.Management.Automation/engine/regex.cs

```diff
@@ -53,6 +53,10 @@ public sealed class WildcardPattern
         // char that escapes special chars
```

Comment:
@iSazonov, your last commit had 1 failures in `PowerShell-CI-windows`
Resources strings in System.Management.Automation (was -ResGen used with Start-PSBuild).'Modules' should be an available type and the strings should be correct
```powershell
Expected strings to be the same, but they were different.
Expected length: 195
Actual length:   204
Strings differ at index 121.
Expected: '...psd1' and ...'
But was:  '...psd1', '.c...'
at <ScriptBlock>, D:\a\1\s\test\powershell\engine\ResourceValidation\TestRunner.ps1: line 64
64:                     NormalizeLineEnd($resourceStringToCheck) | Should -Be (NormalizeLineEnd($inresource.value))
```